pkg/goversion/compat.go

```diff
@@ -10,20 +11,30 @@ var (
 	MaxSupportedVersionOfGoMajor = 1
 	MaxSupportedVersionOfGoMinor = 17
 	goTooOldErr                  = fmt.Errorf("Version of Go is too old for this version of Delve (minimum supported version %d.%d, suppress this error with --check-go-version=false)", MinSupportedVersionOfGoMajor, MinSupportedVersionOfGoMinor)
+	goTooOldWarn                 = fmt.Errorf("Warning: version of Go is too old for this version of Delve (minimum supported version %d.%d)", MinSupportedVersionOfGoMajor, MinSupportedVersionOfGoMinor)
```

Comment:
I don't think the hint about how to disable the error should be removed.

Reply:
The existing hint wording makes it sound like you can disable the warning by using the false value when in fact the false value is what triggers the warning. Unless we want to have more modes with another flag like -- warn-go-version.

Reply:
Ok.